cache/in-memory/src/lib.rs

```diff
@@ -737,9 +737,34 @@ impl InMemoryCache {
 
         let user_id = vs.user_id;
 
+        // Check if the user is switching channels in the same guild (ie. they already have a voice state entry)
+        if let Some(voice_state) = self.0.voice_states.get(&(guild_id, user_id)) {
+            // If the previous voice state existed
+            if let Some(channel_id) = voice_state.channel_id {
+                // Remove the mapping if it already exists and check if the voice states are empty
+                let remove_channel_mapping = self
+                    .0
+                    .voice_state_channels
+                    .get_mut(&channel_id)
+                    .map(|mut channel_voice_states| {
+                        channel_voice_states.remove(dbg!(&(guild_id, user_id)));
```

Comment:
Looks like some debugging slipped in :O 
```suggestion
                        channel_voice_states.remove(&(guild_id, user_id));
```

Reply:
Indeed it did